### PR TITLE
Self-serve convert personal account to organization

### PR DESCRIPTION
**Pull request by bgrenon**
Preview link:
https://neon-next-git-bgrenon-create-org-with-transfer-neondatabase.vercel.app/docs/manage/organizations#create-an-organization

### CODE REVIEW

**Review by PlayLikeNeverB4 — APPROVED** [review]

**Comment by danieltprice** [comment]
Unfortuntaley, the sidebar links do not work if you are on the Free tab:

image

File: content/docs/manage/organizations.md [hunk 1]
```diff
@@ -37,27 +37,72 @@ Here's a breakdown of what actions each role can take:
...
-![create organization button](/docs/manage/orgs_create_button.png)
+### Free Plan
 
-You'll be asked to select a plan and confirm your billing details. After confirming billing, you'll be directed to your organization's billing page, where you can take steps like inviting [members](/docs/manage/orgs-manage#invite-members) and [transferring projects](/docs/manage/orgs-project-transfer).
+If you're on the Free Plan, you'll need to choose a paid plan for your organization (since organizations are a paid feature).
+
+<div style={{ display: 'flex', alignItems: 'top' }}>
+  <div style={{ flex: '0 0 45%', paddingRight: '20px' }}>
+
+    You'll be prompted to choose to select a plan and enter billing details. After confirming, you'll be directed to your organization's billing page, where you can invite [members](/docs/manage/orgs-manage#invite-members) and [transfer projects](/docs/manage/orgs-project-transfer).
```
Comment (danieltprice) [inline content/docs/manage/organizations.md:56]:
```suggestion
    You'll be prompted to select a plan and enter billing details. After confirming, you'll be directed to your organization's billing page, where you can invite [members](/docs/manage/orgs-manage#invite-members) and [transfer projects](/docs/manage/orgs-project-transfer).
```

File: content/docs/manage/organizations.md [hunk 2]
```diff
@@ -37,27 +37,72 @@ Here's a breakdown of what actions each role can take:
...
+  </div>
+</div>
+
+#### Convert your personal account
+
+Converting your personal account transfers all of your projects and billing to the new organization. Your personal account will switch to the Free Plan. This is a quick, one-time, one-way operation: everything in your personal account is transferred to your new organization instantly. There is no service disruption, no change to your connections, and no way to move resources back to your personal account after the conversion.
+
+#### Create a new organization
+
+If you choose to create a new organization, you can decide which projects (if any) you want to [transfer](/docs/manage/orgs-project-transfer). First, you'll need to select a paid plan for the organization. Your personal account will remain on its current paid plan — so note that you'll be billed for both your personal and organization accounts. This option is helpful if you want to keep personal projects separate while still requiring paid features like higher allowances or additional security.
```
Comment (danieltprice) [inline content/docs/manage/organizations.md:87]:
```suggestion
If you choose to create a new organization, you can decide which projects (if any) you want to [transfer](/docs/manage/orgs-project-transfer). First, you'll need to select a paid plan for the organization. Your personal account will remain on its current paid plan — so you'll be billed for both your personal and organization accounts. This option is helpful if you want to keep personal projects separate while still requiring paid features like higher allowances or additional security.
```

File: content/docs/manage/organizations.md [hunk 3]
```diff
@@ -37,27 +37,72 @@ Here's a breakdown of what actions each role can take:
...
 ## What's next?
 
-After creating the organization, your next actions depend on your particular scenario. Here are a couple example scenarios.
+After creating the organization, your next actions depend on your particular scenario. Here are a couple of example scenarios.
 
-### Scenario 1 &#8212; Moving an existing team to a new organization
+### Scenario 1 — Moving an existing team to a new organization
 
-If you’re currently on a paid plan, using project collaboration to work with your team on your Personal account projects, and you intend to work primarily from your new Organization account:
+If you're currently on a paid plan, using project collaboration to work with your team on your Personal account projects, and you intend to work primarily from your new Organization account:
```
Comment (danieltprice) [inline content/docs/manage/organizations.md:99]:
```suggestion
If you're currently on a paid plan, using [project collaboration](/docs/guides/project-collaboration-guide) to work with your team on your Personal account projects, and you intend to work primarily from your new Organization account:
```

File: content/docs/manage/organizations.md [hunk 4]
```diff
@@ -37,27 +37,72 @@ Here's a breakdown of what actions each role can take:
...
+
+#### Create a new organization
+
+If you choose to create a new organization, you can decide which projects (if any) you want to [transfer](/docs/manage/orgs-project-transfer). First, you'll need to select a paid plan for the organization. Your personal account will remain on its current paid plan — so note that you'll be billed for both your personal and organization accounts. This option is helpful if you want to keep personal projects separate while still requiring paid features like higher allowances or additional security.
+
+<Admonition type="note">
+Project transfers to an organization are one-way. You cannot move organization projects back to your personal account.
+</Admonition>
+
 ## What's next?
```
Comment (danieltprice) [inline content/docs/manage/organizations.md:93]: Does this section redundant now that there's self-serve orgs? If you follow the self-serve instructioons, would these next steps still apply?

**Review by danieltprice — APPROVED** [review]
a couple of comments

File: content/docs/manage/organizations.md (hunk 4 above)
Comment (bgrenon) [reply content/docs/manage/organizations.md]: Yes, good point. I've moved relevant content into the main create flow and removed the section.

**Pull request merged**